contrib/test/roles/irc_channel_notice/files/author_to_nick.csv

```diff
@@ -0,0 +1,2 @@
+willgates:cevich
+rhatdan:dwalsh
```

Comment:
I take if if you don't have an entry here, we will use the same name, IE 
runcom -> runcom
mpatel -> mpatel
...

Reply:
Precisely, my thinking is to always pass in ``irc_to`` when referencing the role and...

Reply:
TomSweeneyRedHat:tsweeney

Reply:
This is just for testing, but I can add you in, NP.